reposerver/apiclient/repository.pb.go

```diff
@@ -249,7 +249,7 @@ func (m *TestRepositoryRequest) GetRepo() *v1alpha1.Repository {
 	return nil
 }
 
-// TestRepositoryRequest is a query to test repository is valid or not and has valid access.
+// TestRepositoryResponse represents the TestRepository response
```

Comment:
The repository.pb.go is autogenerated file. Comment shuold be updated in https://github.com/argoproj/argo-cd/blob/dc24380065791244b6e9a11a9fd39ac7e9484137/reposerver/repository/repository.proto#L39